XIV on Mac/ja.lproj/Localizable.strings

```diff
@@ -89,3 +91,45 @@
 "FIRSTAID_CFGCHECK_HBAO_EXP" = "SSAO(Screen Space Ambient Occlusion)はMacでサポートされていません。グラフィックが黒く描画されることがあります。";
 "FIRSTAID_CFGCHECK_TESSELATION_TITLE" = "テッセレーション";
 "FIRSTAID_CFGCHECK_TESSELATION_EXP" = "テッセレーションを「標準」以上に設定すると、Apple Silicon Macで深刻なパフォーマンス低下を引き起こし、視覚的な改善も得られません。";
+
+"SELECT_GAME_PATH_PANEL_TITLE" = "Choose FFXIV install location";
```

Comment:
Waiting on JP localisation from @Umichang.

Reply:
Japanese translation is up.